packages/flutter/lib/src/material/stepper.dart

```diff
@@ -55,6 +55,22 @@ enum StepperType {
   horizontal,
 }
 
+/// Defines the [Stepper]'s titles position.
```

Comment:
Suggestion:

```dart
/// Defines the [Stepper]'s title position when the [type] is [StepperType.horizontal].
```

Reply:
It’s very strange why there is a display error. I confirm that the code is out of date before uploading the code; I will try to reproduce it and fix it。

![WechatIMG36](https://user-images.githubusercontent.com/3699729/126749441-6d079a24-d0b3-4205-8726-42d429af2f5b.jpeg)

Reply:
I don't have this code or branch open right now, but maybe try this without subtitles? I noticed that's different between our samples, and that could be what's causing the discrepancy.

Reply:
Although I have not reproduced the above problem, I have solved a bug that may cause this problem.